ironfish/src/utils/currency.ts

```diff
@@ -58,6 +58,54 @@ export class CurrencyUtils {
     return amount.toString()
   }
 
+  /**
+   * Renders values for human-readable purposes:
+   * - Renders $IRON in the major denomination, with 8 decimal places
+   * - If a custom asset, and `decimals` is provided, it will render the custom
+   *     asset in the major denomination with the decimal places
+   * - If a custom asset, and `decimals` is not provided, it will render the
+   *     custom asset in the minor denomination with no decimal places
+   */
+  static render(
```

Comment:
Thoughts?
```typescript
static render(
    amount: bigint | string,
    includeSymbol: boolean = false,
    assetId?: string,
    verifiedAssetMetadata?: {
      decimals?: number
      symbol?: string
    },
  ): string {
    if (typeof amount === 'string') {
      amount = this.decode(amount)
    }

    let symbol: string
    let decimals: number
    // If an asset ID was provided, check if it is the native asset. Otherwise,
    // we can only assume that the amount is in the native asset
    if (!assetId || isNativeIdentifier(assetId)) {
      // Hard-code the amount of decimals in the native asset
      decimals = IRON_DECIMAL_PLACES
      symbol = '$IRON'
    } else {
      // Default to displaying 0 decimal places for custom assets
      decimals = verifiedAssetMetadata?.decimals || 0
      symbol = verifiedAssetMetadata?.symbol || assetId
    }

    const majorDenominationAmount = FixedNumberUtils.render(amount, decimals)

    if (includeSymbol) {
      return `${symbol} ${majorDenominationAmount}`
    }

    return majorDenominationAmount
  }
```